chalice/cli/__init__.py

```diff
@@ -162,16 +162,19 @@ def delete(ctx, profile, stage):
               default=False,
               help='Controls whether or not lambda log messages are included.')
 @click.option('--stage', default=DEFAULT_STAGE_NAME)
+@click.option('-n', '--name',
+              help='The name of the lambda function to retrieve logs from.',
+              default='api_handler')
```

Comment:
What are your thoughts on using `chalice.constants.DEFAULT_HANDLER_NAME` instead of the hard coded string here?

Reply:
Yeah, good catch.  I'll update before merging.